### ISSUE TITLE
Handle gifs where the width is not evenly divisible by the number of frames

### ISSUE THREAD

**Issue by hschallh** [hschallh/congaaas#2]
**Problem**: [This line]( https://github.com/hschallh/congaaas/blob/master/script#L11) returns an integer. However, the true shift in each frame may be slightly more than the returned shift, leaving a noticeable skip between the last and first frames. See below for an example.

**Steps to reproduce**:
```
cd congaaas
mv example/gif/width_not_divisible_by_frames.gif ./original.gif
./script
```
**Original gif**:
![Original gif](https://raw.githubusercontent.com/hschallh/congaaas/master/example/gif/width_not_divisible_by_frames.gif "original.gif")
The width is 128px and there are 24 frames. This gives a 5px shift for each frame, but leaves the final frame 8px away from where it "should" be.

**Result**:
![Resulting gif](https://raw.githubusercontent.com/hschallh/congaaas/master/example/gif/width_not_divisible_by_frames_result.gif "result.gif")
Notice the skip at the very end of the gif

**Comment by hschallh** [comment]
Closed with 5176253a0994c18b132175747ba80c8960aa99a5

**Comment by hschallh** [comment]
closed with 5176253a0994c18b132175747ba80c8960aa99a5
